pymongo/mongo_client.py

```diff
@@ -1583,6 +1583,11 @@ def _process_kill_cursors(self):
             try:
                 self._cleanup_cursor(True, cursor_id, address, sock_mgr,
                                      None, False)
+            except InvalidOperation as e:
+                if self._topology._closed:
```

Comment:
It's worth adding a comment here to explain why we're raising the error when the client is closed.

Reply:
Done.